### ISSUE TITLE
Add AF3 backend

### ISSUE THREAD

**Issue by abiadak** [KosinskiLab/AlphaPulldown#468]
I suppose it might have already crossed your minds, but do you plan to add the option to make predictions with AF3 instead of AF2?

**Comment by jkosinski** [comment]
Yes, it is being considered to be added as an optional backend (there are many reasons to keep AF2).

**Comment by jkosinski** [comment]
The restrictive license of AF3 models, output predictions, and code and hallucinations in structures. We hope to integrate the OpenFold AF3 clone that would not have the license restrictions, but the AF3 original backend for academic use will still be an option.

**Comment by ebettler** [comment]
Hi,
thank you for your wonderful job with AlphaPullDow. 
For me AF3 should not be a priority (not only for restrictive license):
- for multimer AF3 is only slightly better than AF2 except for antibody/protein complexes
- OpenFold shoud be a priority because it is at least as good as AF2, with a better memory management allowing bigger complexes and will continue to be develop (instead of AF2)
- Chai-1, boltz-1 are opensource, with not restrictive apache license !!

Is it possible to allow other MSA softwares ? like mmseq-gpu or deepmsa2 ?

**Comment by jkosinski** [comment]
Thanks for these good points. Chai-1, boltz-1 are certainly interesting, in particular Chai-1, which can take restraints in input and opened up the license last week. Of course, at some point, handling too many backends will be beyond our capacity so contributions are welcome :-) Also, some testing of the new tools is needed, re overfitting, generalization, etc.

We support MMSeqs2 already:
https://github.com/KosinskiLab/AlphaPulldown?tab=readme-ov-file#13-run-using-mmseqs2-and-colabfold-databases-faster
I suppose MMSeqs2-gpu gives compatible output.

We have no plans for deepmsa2 at the moment.

**Comment by abiadak** [comment]
Thanks a lot for considering it and for proposing other alternative backends! I can do some beta testing with new implementations, albeit on a small scale, due to limited GPU resources.

**Comment by DimaMolod** [comment]
Added in #4d802be7d6525ca991d5e6a9eff749f5b5d6eb2b and AP v.2.1.0
Closing...
